Bottom: 5332c6317bea6398fa7fc5e2053cba0024d4b1ed
Top:    40597db6e40358ffa6dc2e48886d8fea779edd56
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-17 02:49:40 +0100

Avoid clearing pins / vias from polygons... it is already done!




---

diff --git a/src/draw.c b/src/draw.c
index 9c6d8ee..d52515f 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1238,6 +1238,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
       AddPart (Pin);
       return;
     }
+#if 0
   /* Clear the area around the pin */
   if (TEST_FLAG (SQUAREFLAG, Pin))
     {
@@ -1260,6 +1261,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
     {
       gui->fill_circle (Output.pmGC, Pin->X, Pin->Y, half);
     }
+#endif
   if ((!TEST_FLAG (PINFLAG, Pin) && !PCB->ViaOn)
       || (TEST_FLAG (PINFLAG, Pin) && !PCB->PinOn))
     return;
